# map_server_params.yaml
map_server:
  ros__parameters:
    frame_id: map
    topic_name: map
    use_sim_time: false
    yaml_filename: "/maps/map7.yaml"  
